### PR TITLE
DM-31447: publish motors.current telemetry

### PR DESCRIPTION
**Pull request by r-owen**
Please do NOT merge this PR. It would be much better to cherry-pick the few relevant changes when the new low-level controller is fully implemented and ready to use.

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -6,6 +6,23 @@
...
 ###############
 
+v0.24.1
+-------
+
+Changes:
+
+* Stop publishing motors.calibrated telemetry.
+  The new low-level controller won't provide the necessary information and we want to remove it from the XML.
+* Restore output of motor current.
```
Comment (teweitsai) [inline doc/version_history.rst:16]: I thought we need to publish the bus current as well.

File: python/lsst/ts/mtrotator/mock_controller.py [hunk 2]
```diff
@@ -320,6 +320,9 @@ async def update_telemetry(self, curr_tai):
             self.telemetry.current_pos = curr_pos
             self.telemetry.current_vel_ch_a_fb = curr_segment.velocity
             self.telemetry.current_vel_ch_b_fb = curr_segment.velocity
+            self.telemetry.motor_current[:] = [motor_current] * 2
+            self.telemetry.bus_voltage = 20  # arbitrary
+            # These fields seem to duplicate telemetry.motor_current?
             self.telemetry.motor_current_axis_a = motor_current
             self.telemetry.motor_current_axis_b = motor_current
```
Comment (teweitsai) [inline python/lsst/ts/mtrotator/mock_controller.py:327]: What is the difference among the `self.telemetry.motor_current[:]`, `self.telemetry.motor_current_axis_a`, and `self.telemetry.motor_current_axis_b`?

**Review by teweitsai — APPROVED** [review]
Looks good. I can see the use of bus voltage and currents in the C structure now. But I should admit I did not check all the code to make sure the bus voltage is published to EFD or not. I only see the publish of current in this PR. If the bus voltage is published already, please just ignore me. Thanks!

File: python/lsst/ts/mtrotator/mock_controller.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/mtrotator/mock_controller.py:327]: I reply this by myself. They are the relationship of outputs and inputs:
1. https://github.com/lsst-ts/ts_rotator_controller/blob/develop/include/interface/telemetryStructure.h#L101-L102
2. https://github.com/lsst-ts/ts_rotator_controller/blob/develop/include/interface/telemetryStructure.h#L183-L192

Thanks!

Comment (r-owen) [reply python/lsst/ts/mtrotator/mock_controller.py:327]: I was asking you that question! But you answered it on Slack. You can see how these values map to the telemetry struct by looking at the structs.py file. It turns out motor_current is measured motor current, whereas the motor_current_axis_a/b fields are demand motor current.

**Pull request closed without merging**